Add clinical notes (#80)
nckhell/awell-developers: content/awell-orchestration/api-reference/queries/get-clinical-note.mdx

---
title: Get clinical note activity
description: Get the generated clinical note activity with a given id
---

> This query allows us to fetch the content for a specific clinical note activity.

Reviewer: ```suggestion
This query allows us to fetch the details of a specific clinical note activity.
```

## Request

### Query

```graphql
query GetClinicalNote($id: String!) {
  clinicalNote(id: $id) {
    clinical_note {
      id
      narratives {
        id
        key
        title

>         body

Reviewer: ```suggestion
        body  // a string containing html content
```
